apps/pwa/pages/trips/[id].tsx

```diff
@@ -30,6 +31,29 @@ export default function TripById() {
   // the trip ID
   const id = router.query.id as string
 
+  useEffect(() => {
```

Comment:
i think instead of using an effect to fetch data, we can use the useFirestoreConnect() hook to grab the users labels, and then that will automatically be in the redux store

Reply:
I do like the sound of it, but I attempt haven't worked out too well. Right now, deleting or creating a new label (for user) doesn't update the store so the list looks always the same. Any idea where I might be wrong?